lib/core/print.toit

```diff
@@ -36,22 +36,78 @@ Does not yield the currently running task.
 debug:
   print_
 
-// Dumps an empty line on stdout.
+/**
+Dumps the an empty new line on stdout and flushes it.
+
+Does not yield the currently running task.
+*/
 print_:
   print_ ""
 
-// Dumps print string of $object on stdout.
+/**
+Dumps the string of $object and a newline on stdout and flushes it.
+
+Does not yield the currently running task.
+*/
 print_ object:
-  print_string_on_stdout_ object.stringify
+  write_string_on_stdout_ object.stringify true
+
+/**
+Dumps the string $message and a newline on stdout and flushes it.
 
-// Dumps the string $message on stdout and flushes.
+Does not yield the currently running task.
+*/
 print_string_on_stdout_ message:
-  #primitive.core.print_string_on_stdout
+  write_string_on_stdout_ message true
+
+/**
+Dumps the string of $object on stdout and flushes it.
+If $add_newline is true adds a "\n" to the output.
+
+Does not yield the currently running task.
+*/
+write_on_stdout_ object add_newline :
```

Comment:
Remove space before :.

Reply:
Is it worth having this extra indirection instead of just letting the two callers call stringify?

Reply:
Removed the space.

Will do a follow-up PR to remove the indirections.